### DISCUSSION TITLE
add OVH certbot dns challenge to nginxproxymanager

### DISCUSSION THREAD

**Discussion by jsapede** [tteck/Proxmox#2616] [Help Needed]
Hello,
i'm not familiar with this, but as far as i understand i need to install OVH dns challenge for certbot on NPM.
do you simply do it by

 `pip install certbot-dns-ovh` 

? without any venv considerations ? 
where are stored the configs to edit for those challenges ? 

`dns_ovh_endpoint = ovh-eu
dns_ovh_application_key = xxx
dns_ovh_application_secret = xxx
dns_ovh_consumer_key = xxx`

i've explored the installation script before asking but didnt find

**Comment by JC38** [comment]
python3 -m pip install certbot-dns-ovh was the solution for me to add the OVH feature to certbot.

**Comment by burgerga** [comment]
Or select DnsMulti from the menu, `certbot-dns-multi` is already installed by tteck, and it has support for OVH
